src/rprblender/properties/view_layer.py

```diff
@@ -247,162 +248,194 @@ class RPR_ViewLayerProperites(RPR_Properties):
         {
             'rpr': pyrpr.AOV_COLOR,
             'name': "Combined",
-            'channel': 'RGBA'
+            'channel': 'RGBA',
+            'hybridpro': True
```

Comment:
instead of adding 'hybridpro' option better all list variable with unsupported AOVs in pyhybridpro.py